ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts

```diff
@@ -37,29 +37,44 @@ import { Router } from '@angular/router';
       *ngIf="
         $any(results$ | async)?.length > 0 && (isLoading$ | async) === false
       "
-      ><ess-pagination
-        [paginationData]="$any(paginationData$ | async)"
-        [loading]="(isLoading$ | async) ?? false"
-        (activePageChange)="pageNr$.next($event)"
-      ></ess-pagination>
-      <ess-result
-        class="results"
+    >
+      <ng-container
         *ngFor="let result of results$ | async; trackBy: trackByResultId"
-        [id]="result.id"
-        [title]="result.title"
-        [description]="result.description"
-        [type]="result.type"
-        [url]="result.url"
-        [tags]="result.tags"
-        [coloredTags]="result.coloredTags ?? []"
-        [secondaryTags]="result.secondaryTags ?? []"
-        [views]="result.views"
-        [downloads]="result.downloads"
-        [accessRight]="result.accessRight"
-        [date]="result.date"
-        [highlights]="highlights[result.id] ?? {}"
-        [offers]="result.offers ?? []"
-      ></ess-result>
+      >
+        <ess-result
+          class="results"
+          *ngIf="!result.type.value.toString().includes('provider')"
```

Comment:
This is probably more complicated than it needs to be.

Also in this case either use `if; else` or `switch`. Creating opposite ifs can lead to bugs where both (or none) cases are displayed.

Reply:
I used the else if template